gdx/src/com/badlogic/gdx/math/Vector2.java

```diff
@@ -363,6 +371,12 @@ public float angleRad (Vector2 reference) {
 		return (float)Math.atan2(reference.crs(this), reference.dot(this));
 	}
 
+	/** @return the angle in radians of this vector (point) relative to the x-axis. Angles are towards the positive y-axis.
+	 *         (typically counter-clockwise) */
+	public static float angleRad (float x, float y) {
```

Comment:
I'd say that many many peaople will not know that `(float)Math.atan2(y, x);` will return an angle in radians.

and there are already two non-static implementations of `public float angleRad` in `Vector2 `with exaclty this code. Should they be removed then as well?

Reply:
Many people don't know what radians are... nor degrees... does that mean we can't use them?

I'm only familiar with angle() in Vector2 from when it was incorrectly implemented and returned results for negative y instead of the given y. A fair amount of my code depends on older libGDX versions for compatibility and so actually cannot use the Vector2 angle methods without having very different behavior based on libGDX version.

I think we need to be careful with what we add to Vector2 because it can't easily be changed, and we definitely can't remove code that's been in a release. I'm really unsure why Vector2 uses Math and casting when we have MathUtils available for float trigonometry methods.